src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { KeyboardEvent, useState } from "react";
+import "./index.css";
+
+const QuestionMarkTooltip = ({ description }: { description: string }) => {
+  const [show, setShow] = useState<boolean>(false);
+
+  return (
+    <div
+      onKeyDown={(e: KeyboardEvent<HTMLDivElement>) => {
+        if (e.nativeEvent.key === "Escape") {
+          setShow(false);
+        }
+      }}
+      style={{ position: "relative" }}
+    >
+      <button
+        onClick={() => {
+          setShow((prev) => !prev);
+        }}
+        className="tooltip"
+        aria-label="성인 승객 관련 정보"
+      >
+        ?
+      </button>
+      {show && (
+        <div className="tooltip-description" aria-live="polite">
+          {description}
+        </div>
```

Comment:
tooltip 메시지가 표시되는 요소를 위한 `tooltip` [aria role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)이 있습니다.

```suggestion
        <div role="tooltip" className="tooltip-description" aria-live="polite">
          {description}
        </div>
```